posthog/api/test/test_event.py

```diff
@@ -151,6 +152,71 @@ def test_filter_by_person(self):
         response = self.client.get(f"/api/projects/{self.team.id}/events/?person_id={person.uuid}").json()
         self.assertEqual(len(response["results"]), 2)
 
+    @mock.patch("posthog.models.team.Team.actor_on_events_querying_enabled", return_value=True)
+    def test_filter_by_person_for_person_on_events_enabled(self, mock_actor_on_events_querying_enabled):
```

Comment:
Would be great if we had query snapshot tests for this to see whether the generated SQL makes sense :thinking:

Reply:
Have added a snapshot for this now 🙏